cluster/gce/gci/mounter/mounter.go

```diff
@@ -0,0 +1,93 @@
+/*
```

Comment:
Ideally, we would maintain this mounter infrastructure outside of core k8s repo to avoid confusion. I'd recommend moving it out in a separate PR